test/msgpack_protocol_test.py

```diff
@@ -1,3 +1,17 @@
+# Copyright (c) 2014-2017 Spotify AB
```

Comment:
This code was introduced in 2014 together with the msgpack implementation. It looks a bit like backdating the copyright but it's just reflecting the reality.